Tests/tbDEXTests/Protocol/Models/Messages/OrderStatusTests.swift

```diff
@@ -5,69 +5,60 @@ import XCTest
 
 final class OrderStatusTests: XCTestCase {
     
-    let did = try! DIDJWK.create(keyManager: InMemoryKeyManager())
-    let pfi = try! DIDJWK.create(keyManager: InMemoryKeyManager())
-
-    func test_init() {
-        let orderStatus = createOrderStatus(from: did.uri, to: pfi.uri)
-
-        XCTAssertEqual(orderStatus.metadata.id.prefix, "orderstatus")
-        XCTAssertEqual(orderStatus.metadata.from, did.uri)
-        XCTAssertEqual(orderStatus.metadata.to, pfi.uri)
-        XCTAssertEqual(orderStatus.metadata.exchangeID, "exchange_123")
-        XCTAssertEqual(orderStatus.data.orderStatus, "test status")
+    func test_parseOrderStatusFromStringified() throws {
+        if let orderStatus = try parsedOrderStatus(orderStatus: orderStatusStringJSON) {
+            XCTAssertEqual(orderStatus.metadata.kind, MessageKind.orderStatus)
+        } else {
+            XCTFail("Order status is not a parsed orderStatus")
```

Comment:
i want to understand how you expect this test to fail - are we calling this test function with 1 valid orderstatusJson and expecting it to never drop into the `else` assertion here? 

lining up multiple test cases and running them through tests to ensure what we want to see pass passes and what we want to see fail fails is how we've been approaching it